examples/gno.land/r/sys/validators/v2/poc.gno

```diff
@@ -37,7 +33,9 @@ func NewPropExecutor(changesFn func() []validators.Validator) dao.Executor {
 		return nil
 	}
 
-	return bridge.GovDAO().NewGovDAOExecutor(callback)
+	e := dao.NewSimpleExecutor(callback, "")
+
+	return dao.NewProposalRequest(title, description, e)
```

Comment:
```suggestion
	return dao.NewSimpleExecutor(callback, "")
```

for composable proposals, we need to return the executor, not a proposal.

creating a proposal may eventually be templated, but we expect them to be made in maketx run or other mechanisms while we expect this contract to provide a templated safe executor (!= proposal).

Reply:
We are not returning a Proposal, we return a ProposalRequest, that makes the entire object safe (Title, Description, Executor, everything is secure when a ProposalRequest is created).